### ISSUE TITLE
Make corpus snapshots more often in the first few mins

### ISSUE THREAD

**Issue by lszekeres** [google/fuzzbench#656]
We could make snapshots according to some reciprocal function, ie. a lot in the beginning and then less and less.

**Comment by inferno-chromium** [comment]
More in the early stage +1, unsure if we need to reduce frequency later.

**Comment by mboehme** [comment]
Yeah, given the exponential cost of increasing coverage, it makes more sense to capture in exponentially increasing intervals. i.e., $c^i$ where $c$ is some constant and $i \in [0, N]$ such that c^N = 24 hours. On a log-scale, this should come out as equally spaced data points.

**Comment by lszekeres** [comment]
or we could keep doing snapshots every few minutes, but use timestamps to reconstruct creation time as suggested in https://github.com/google/fuzzbench/issues/715

**Comment by mboehme** [comment]
Nice!
